lib/internal/Magento/Framework/Url.php

```diff
@@ -478,8 +483,9 @@ public function getBaseUrl($params = [])
         ) {
             $this->getRouteParamsResolver()->setType(UrlInterface::URL_TYPE_DIRECT_LINK);
         }
-
-        $result = $this->_getScope()->getBaseUrl($this->_getType(), $this->_isSecure());
+        $customScope
+            ? $result = $this->getData('scope')->getBaseUrl($this->_getType(), $this->_isSecure())
+            : $result = $this->_getScope()->getBaseUrl($this->_getType(), $this->_isSecure());
```

Comment:
Who taught you this?)

Please use ternary operator normally, `$result = $customScope ? ... : ...`

Reply:
I'm just thinks that this more clearly, do i need to use "if else" ? at this point ?

Reply:
Still not fixed.